PsychSourceGL/Source/Common/IOPort/IOPort.c

```diff
@@ -369,7 +369,8 @@ PsychError IOPORTOpenSerialPort(void)
         "or on OS/X and Linux in 'Cooked' processing mode as line delimiter. A setting of -1 will try to disable the line terminator.\n\n"
         "DTR=os default    -- Setting for 'Data Terminal Ready' pin: 0 or 1.\n\n"
         "RTS=os default    -- Setting for 'Request To Send' pin: 0 or 1.\n\n"
-        "BreakBehaviour=Ignore -- Behaviour if a 'Break Condition' is detected on the line: Ignore, Flush, Zero. On Windows, this setting is ignored.\n\n"
+        "BreakBehaviour=Ignore -- Behaviour if a 'Break Condition' is detected on the line: Ignore, Flush, Zero. On Windows, this "
```

Comment:
Hmm, thinking further about this: I think we should not "Ignore" the setting, instead we should validate the parameter and error abort with a "Setting not supported on Windows" error message or such, if a setting other than `Ignore` is specified.

This way the default or explicit setting of `Ignore` will do the right thing - and compatible to other OS'es, but users get a heads up if they try to ask for a behaviour that is fine on Linux/macOS, but unsupported on Windows. I think that would be the most sane approach, and the one I should have implemented in the first place.

PsychSerialWindowsGlue.c already has the commented out if-else statements, so one just has to uncomment and update it accordingly, and change the last sentence of the help text here to "On Windows, only 'Ignore' is allowed."